com.github.AmatCoder.mednaffe.appdata.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally you should maintain it in your own repo.

Reply:
Yes, but I would like to give permission to edit it to mednafen author.